apps/web/modules/d/[link]/d-type-view.tsx

```diff
@@ -5,6 +5,8 @@ import { getBookerWrapperClasses } from "@calcom/features/bookings/Booker/utils/
 
 import { type PageProps } from "@lib/d/[link]/[slug]/getServerSideProps";
 
+import BookingPageErrorBoundary from "@components/error/BookingPageErrorBoundary";
```

Comment:
Client-side errors thrown in `Booker` will now get caught in `BookingPageErrorBoundary`

Reply:
Errors thrown in RSCs will render:
<img width="1512" alt="Screenshot 2025-04-01 at 5 20 33 PM" src="https://github.com/user-attachments/assets/59cdad77-0b2b-47e6-a2e4-16cb815c0eee" />

While client-side errors caught in BookingPageErrorBoundary will render:
<img width="1508" alt="Screenshot 2025-04-01 at 5 10 03 PM" src="https://github.com/user-attachments/assets/5e89fa0f-96e7-4273-9cf8-0fea2723e0ea" />
<img width="1507" alt="Screenshot 2025-04-01 at 5 10 06 PM" src="https://github.com/user-attachments/assets/5a719222-3ada-406b-b306-88ea61ef6852" />

Reply:
@hbjORbj How does it look in a production build?